api/pyproject.toml

```diff
@@ -35,7 +35,7 @@ dependencies = [
   "python-decouple >=3.8, <4",
   "python-xmp-toolkit >=2.0.2, <3",
   "sentry-sdk >=1.40.6, <2",
-  "uvicorn[standard] >=0.27.1, <0.28",
+  "uvicorn[standard] >=0.29, <0.30",
```

Comment:
Well, Renovate decided to drop the patch component. It's unfortunate, but not a problem.

Reply:
They're identical, at least according to https://devhints.io/semver

When the left-hand/lower-bound side is partial, it's presumed to finish with `.0`, so `0.29` is identical to `0.29.0`.